lib/vSphere/util/vim_helpers.rb

```diff
@@ -13,12 +13,22 @@ def get_vm_by_uuid(connection, machine)
         end
 
         def get_resource_pool(datacenter, machine)
-          compute_resource = get_compute_resource(datacenter, machine)
-          rp = compute_resource.resourcePool
-          unless machine.provider_config.resource_pool_name.nil?
-            rp = compute_resource.resourcePool.find(machine.provider_config.resource_pool_name)
-            fail Errors::VSphereError, :missing_resource_pool if rp.nil?
+          rp = get_compute_resource(datacenter, machine)
+          entity_array = machine.provider_config.resource_pool_name.split('/')
+          entity_array.each do |entity_array_item|
+            puts entity_array_item
```

Comment:
This must look odd in the output. No news is good news. We shouldn't be outputting each item in the array